openrarity/collection/utils.py

```diff
@@ -8,23 +8,25 @@
 from openrarity.token.types import TokenId, TokenStatistic
 
 
-def flatten_token_data(
+def   flatten_token_data(
```

Comment:
Remove the extra space. Are you running the pre-commit formatters?

Reply:
started using black.